File: Angular-app/src/app/modules/currency-rates/models/currency-rate-value.ts
export class CurrencyRateValue {
	constructor(currencyRates: Partial<CurrencyRateValue>) {
		this.updateDate = currencyRates.updateDate;
		this.officialRate = currencyRates.officialRate;
		this.ratePerUnit = currencyRates.ratePerUnit;
	}

	ratePerUnit?: number;
	updateDate?: Date;
	officialRate?: number;
}

File: Api/HomeBudget-Rates-Api/HomeBudget.Components.CurrencyRates/Models/CurrencyRateValue.cs
using System;

namespace HomeBudget.Components.CurrencyRates.Models
{
    public class CurrencyRateValue
    {
        public decimal OfficialRate { get; set; }

        public decimal RatePerUnit { get; set; }

        public DateOnly UpdateDate { get; set; }
    }
}
